docs/api/browser-view.md

```diff
@@ -94,6 +94,10 @@ Returns `Boolean` - Whether the view is destroyed.
 
 Resizes and moves the view to the supplied bounds relative to the window.
 
+#### `view.getBounds()` _Experimental_
+
+Returns [`Rectangle`](structures/rectangle.md)
```

Comment:
For extra context, we might say:
```
Returns [`Rectangle`](structures/rectangle.md)

The `bounds` of this BrowserView instance as `Object`.
```

Reply:
Either way it should be consistent with [BrowserWindow#getBounds](https://github.com/electron/electron/blob/master/docs/api/browser-window.md#wingetbounds).

Reply:
I would prefer we update it there as well in that case, since it's the case for [Tray bounds](https://github.com/electron/electron/blob/24673501804bc372b73577896cf00cb6282bf667/docs/api/tray.md#L280-L284) and for documentation, more verbosity is generally preferable to less within reason.

Reply:
oki, then i'll add it on both docs later